### ISSUE TITLE
Update to match Cairn 11-25-2020

### ISSUE THREAD

**Issue by yochaigal** [yochaigal/Cairn-FoundryVTT#5]
TL; DR: Gold, Rations & Torches now fit in one slot on each PC. No more Party Inventory for these resources.

Background tables should be checked, every other relevant is listed below.

Text Changes: Typos and inconsistent text were fixed throughout.

Character creation: 
PCs now start with a torch and a week's Rations. Both share a slot with Gold.
Torches are equipped (in the fiction and/or on the sheet) by being moved to a hand-slot.
Party Inventory has been limited to only Climbing Gear and Camping Gear.
Additional information has been added regarding Horses (2 slots), Mules (4 slots) and Wagons (8 slots).

Traits:
A few new backgrounds have been added to further apply the overall theme.
Reputation: Lazy replaced with Loafer

Starting Gear:
Tools: Waterskin has been replaced with Poisoner's Kit
Bonus Items are now Tool or Trinket, Exped. Gear, Armor or Weapon, Spellbook
Optional Gear Packages: updated items to better reflect each trope

Equipment List: 
Adjusted prices to more closely match B/X and OSE.
Added Horse & Mule (with slots), and removed bulky properties from Rations & Torches.

Rules:
Healing: clarified magical healing.
Armor: made Armor consistent throughout (e.g. N Armor)
Hirelings: made them more like PCs, especially their Ability Scores and inventory limits.
Casting Spells: clarified Fatigue rules.
Combat: cleaned up a bit.
Scars: reworked almost every one of them. Most improve the PC, after recovery.
